packages/snaps-execution-environments/scripts/build.js

```diff
@@ -127,6 +133,19 @@ async function main() {
       bundler.transform(require('@browserify/uglifyify'), { global: true });
       bundler.plugin(require('common-shakeify'), { ecmaVersion: 2020 });
 
+      let extraOptions = {};
+
+      if (worker) {
+        extraOptions = {
```

Comment:
Perhaps define as `lavamoatSecurityOptions` and re-use between iframe and worker builds?